.gitignore

```diff
@@ -12,3 +12,6 @@ phpcs.xml
 yarn.lock
 docker-compose.override.yml
 /wordpress
+
+playground/dist
+.cache
```

Comment:
Does this need to be in `.eslintignore` ? Ignored tests (#14997) ? I worry its presence could add some significant folder-scan overhead for some common tasks.

Reply:
Is there a config for .eslintignore to automatically include .gitignore?

Reply:
> Is there a config for .eslintignore to automatically include .gitignore?

Not that I'm aware of, though I do sympathize with the goal to try to consolidate how we consider these "ignored" files.

Related: https://github.com/eslint/eslint/issues/5848